pyvista/core/grid.py

```diff
@@ -159,33 +159,35 @@ def _from_arrays(self, x, y, z):
 
     @property
     def meshgrid(self):
-        """Return the a meshgrid of numpy arrays for this mesh.
+        """Return a meshgrid of numpy arrays for this mesh.
 
         This simply returns a ``numpy.meshgrid`` of the coordinates for this
-        mesh in ``ij`` indexing.
+        mesh in ``ij`` indexing. These are a copy of the points of this mesh.
 
         """
         return np.meshgrid(self.x, self.y, self.z, indexing='ij')
 
     @property
     def points(self):
-        """Return all of the points as an n by 3 numpy array."""
+        """Return a copy of the points as an n by 3 numpy array."""
         xx, yy, zz = self.meshgrid
         return np.c_[xx.ravel(order='F'), yy.ravel(order='F'), zz.ravel(order='F')]
 
     @points.setter
     def points(self, points):
-        """Set points without copying."""
-        if not isinstance(points, np.ndarray):
-            raise TypeError('Points must be a numpy array')
-        # get the unique coordinates along each axial direction
-        x = np.unique(points[:,0])
-        y = np.unique(points[:,1])
-        z = np.unique(points[:,2])
-        # Set the vtk coordinates
-        self._from_arrays(x, y, z)
-        #self._point_ref = points
-        self.Modified()
+        """Points must be set along each axial direction.
+
+        Please set the point coordinates with the ``x``, ``y``, and ``z``
+        setters.
+
+        This setter overrides the base class' setter to ensure a user does not
```

Comment:
```suggestion
        This setter overrides the base class's setter to ensure a user does not
```
I had to look this up: https://english.stackexchange.com/a/44085

Reply:
Ahhh thanks! My grade school English teacher would be disappointed in me